plugins/techdocs/src/reader/transformers/html/hooks/hooks.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { removeUnsafeLinks } from './links';
+export { removeUnsafeIframes } from './iframes';
```

Comment:
Why is this file needed? Could be exported from `index.ts`?

Reply:
I started everything in a single file and then extract it into other separate ones, it ended up only with the import lines, I will remove it, thanks for the good eyes on it!